.teamcity/src/main/kotlin/configurations/FunctionalTest.kt

```diff
@@ -64,7 +67,13 @@ class FunctionalTest(
     }
 })
 
-fun enableExperimentalTestDistribution(testCoverage: TestCoverage, subprojects: List<String>) = testCoverage.os == Os.LINUX && (subprojects == listOf("core") || subprojects == listOf("dependency-management"))
+fun enableCheckLinksParameter(testCoverage: TestCoverage, subprojects: List<String>) = if (testCoverage.isPlatform && subprojects.contains("docs")) "-PenableCheckLinks=false" else ""
```

Comment:
Can we encode this logic on our build instead of needing to pass the property?

Reply:
Better now?

Reply:
I think so.